tools/introspection/CMakeLists.txt

```diff
@@ -21,6 +21,12 @@ set(IOX_VERSION_STRING "2.90.0")
 
 project(iceoryx_introspection VERSION ${IOX_VERSION_STRING})
 
+# Do not build the introspection on Windows due to missing ncurses
```

Comment:
Another reason to have a simple `ros topic list` like introspection alongside a more sophisticated one.